.travis.yml

```diff
@@ -17,9 +17,9 @@ env:
     # use_custom_libcxx=false and use_sysroot=false seem to be required to build on
     # Ubuntu 14.04
     # Help: How do you wrap long lines here?
-    - DENO_BUILD_ARGS="is_debug=false use_allocator=\"none\" use_custom_libcxx=false use_sysroot=false"
-    - DENO_BUILD_PATH=$HOME/out/Default
-    - DENO_BUILD_MODE=debug
+    - DENO_BUILD_ARGS="use_custom_libcxx=false use_sysroot=false"
+    - DENO_BUILD_PATH=$HOME/out/release
+    - DENO_BUILD_MODE=release
```

Comment:
I think we want to leave it at 'debug' for running the tests, so that DCHECKs are active etc.
To build the release you could add a separate configuration that only runs on tags.
Or set up cron and build the release build only if `TRAVIS_BRANCH==master` and `TRAVIS_EVENT_TYPE==cron` and `TRAVIS_PULL_REQUEST==false`

Reply:
I have taken this into account, I think, in #501 - the `release` stage will make a new build to publish